software/oada_upload/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
```

Comment:
Since we are targeting ts-node, we should use `ES6` here and use real modules.

Reply:
Could you explain this a little more? Why we want ES6 instead of commonjs? Simply switching that line leads to all sorts of compiler errors

Reply:
Sorry. This is a node script. Been doing too much UI stuff lately. `commonjs` is right